integration-tests/bats/replication.bats

```diff
@@ -160,22 +160,6 @@ teardown() {
     [[ "$output" =~ "t1" ]] || false
 }
 
-@test "replication: push on branch table update" {
```

Comment:
I assume there's a test for creating a new branch with `call dolt_branch()`? That is why this is deleted?

Reply:
`UPDATE dolt_branches ...` query can only be replicated with `CALL DOLT_COMMIT()` in this test. There is already a test below that tests this test with `CALL DOLT_COMMIT()`.

Reply:
Then tests look good to me :-)